tests/e2e/cucumber/features/smoke/share.oc10.feature

```diff
@@ -44,8 +44,8 @@ Feature: share
     #  | folder_to_shared/lorem_new.txt |
     #  | folder_to_shared/simple.pdf    |
     And "Alice" uploads the following resource
-      | resource          | to               | create_version |
-      | PARENT/simple.pdf | folder_to_shared | true           |
+      | resource          | to               | option  |
+      | PARENT/simple.pdf | folder_to_shared | Replace |
     #Then "Alice" should see that the resource "folder_to_shared/simple.pdf" has 1 version
```

Comment:
what will happen to this setp if `create_version` is not set true?

Reply:
if create_version is set to false then Replace will occur. If create_version is not set anything then null will be passed